campaign_themes/arctic.json

```diff
@@ -79,11 +79,12 @@
 			"configurations": {
 				"minimal": {
 					"options": [
-						{ "color":  "#ffffff"},
-						{ "color":  "#000000"},
-						{ "color": "#035880"},
-						{ "color": "#00b6c2"}
-					]
+						{ "value": "#ffffff" },
+						{ "value": "#000000" },
+						{ "value": "#035880" },
+						{ "value": "#00b6c2" }
+					],
+					"default": "#035880"
```

Comment:
Not necessarily something we should do in this PR, but maybe it would be nicer to maintain if we marked default values by setting a `default` property on the object of that option.
```
					"options": [
						{ "value": "#ffffff" },
						{ "value": "#000000" },
						{ "value": "#035880", "isDefault": true },
						{ "value": "#00b6c2" }
					],
```